Documentation/Storage-Configuration/Shared-Filesystem-CephFS/filesystem-storage.md

```diff
@@ -220,6 +220,168 @@ If the Rook cluster has more than one filesystem and the application pod is sche
+spec:
+  capacity:
+    storage: 100Gi
+  csi:
+    driver: rook.cephfs.csi.ceph.com
+    volumeHandle: >-
+      0001-0011-rook-0000000000000001-8a528de0-e274-11ec-b069-0a580a800213-newnamespace
+    volumeAttributes:
+      clusterID: rook
+      fsName: rook-cephfilesystem
+      storage.kubernetes.io/csiProvisionerIdentity: 1654174264855-8081-rook.cephfs.csi.ceph.com
+      subvolumeName: csi-vol-8a528de0-e274-11ec-b069-0a580a800213
+      subvolumePath: >-
+        /volumes/csi/csi-vol-8a528de0-e274-11ec-b069-0a580a800213/da98fb83-fff3-485a-a0a9-57c227cb67ec
+      rootPath: >-
+        /volumes/csi/csi-vol-8a528de0-e274-11ec-b069-0a580a800213/da98fb83-fff3-485a-a0a9-57c227cb67ec
+      staticVolume: "true"
+    nodeStageSecretRef:
+      name: rook-csi-cephfs-node
+      namespace: rook
+    controllerExpandSecretRef:
+      name: rook-csi-cephfs-provisioner
+      namespace: rook
+  accessModes:
+    - ReadWriteMany
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: rook-cephfs
+  volumeMode: Filesystem
+```
+
+* In a new or other namespace, create a new PVC that will use this new PV you created. You simply have to point to it in the `volumeName` parameter. Make sure you enter the same size as the original PVC!:
+
+```yaml
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: second-pvc
+  namespace: newnamespace
+  finalizers:
+    - kubernetes.io/pvc-protection
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 100Gi
+  volumeName: pvc-a02dd277-cb26-4c1e-9434-478ebc321e22-newnamespace
+  storageClassName: rook-cephfs
+  volumeMode: Filesystem
+```
+
+You have now access to the same CephFS subvolume from different PVCs in different namespaces. Redo the previous steps (copy PV with a new name, create a PVC pointing to it) in each namespace you want to use this subvolume.
+
+### Shared volume removal
+
+As the same CephFS volume is used by different PVCs/PVs, you must proceed very orderly to remove it properly.
+
+* Delete the "cloned" PVCs in the different namespaces, but keep the original one!
+* Delete the corresponding "cloned" PVs that should now have been marked as "Released". Again, don't delete the original one yet!
+* Edit the original PV, changing back the `persistentVolumeReclaimPolicy` from `Retain` to `Delete`.
+* Delete the original PVC. It will now properly delete the original PV, as well as the subvolume in CephFS.
+
+### Pending Issue
+
+Due to [this bug](https://github.com/ceph/ceph-csi/issues/2238), the global mount for a Volume that is mounted multiple times on the same node will not be unmounted. This does not result in any particular problem, apart from polluting the logs with unmount error messages, or having many different mounts hanging if you create and delete many shared PVCs, or you don't really use them.
+
+Until this issue is solved, either on the Rook or Kubelet side, you can always manually unmount the unwanted hanging global mounts on the nodes.
```

Comment:
add a step to manually unmount the volume?

Reply:
Procedure added